include/aca_ovs_control.h

```diff
@@ -0,0 +1,199 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACA_OVS_CONTROL_H
+#define ACA_OVS_CONTROL_H
+
+#define IPTOS_PREC_INTERNETCONTROL 0xc0
+#define DSCP_DEFAULT (IPTOS_PREC_INTERNETCONTROL >> 2)
+#define STDOUT_FILENO   1   /* Standard output.  */
+
+#include "goalstateprovisioner.grpc.pb.h"
+#include <openvswitch/vconn-provider.h> /* add to /usr/local/include/openvswitch */
+#include <openvswitch/ofpbuf.h>
+#include <openvswitch/ofp-errors.h>
+#include <openvswitch/ofp-packet.h>
+#include <openvswitch/ofp-port.h>
+#include <openvswitch/ofp-table.h>
+#include <openvswitch/ofp-protocol.h>
+#include <openvswitch/ofp-switch.h>
+#include <linux/in.h>
+#include <string>
+
+extern "C" { 
+    struct unixctl_conn;
+}
+// OVS monitor implementation class
+namespace aca_ovs_control
+{
+class ACA_OVS_Control {
+  public:
+  static ACA_OVS_Control &get_instance();
+
+/* --names, --no-names: Show port and table names in output and accept them in
+ * input.  (When neither is specified, the default is to accept port names but,
+ * for backward compatibility, not to show them unless this is an interactive
+ * console session.)  */
+  static int use_names;
+  static int verbosity;
+  /* -F, --flow-format: Allowed protocols.  By default, any protocol is allowed. */
+  static enum ofputil_protocol allowed_protocols;
+  /* --unixctl-path: Path to use for unixctl server, for "monitor" and "snoop"
+     commands. */
+  char *unixctl_path;
+
+  int control();
```

Comment:
Let us add some comments to the APIs as they appear to be the major interface in this PR.

Reply:
Added some comments for major function calls.